spotifyPlayer.py

```diff
@@ -65,7 +65,7 @@ def playMP3(fileName):
 
 		# Removes the MP3 file once song is done playing
 		def checkPlayStatus(fileName):
-			while pygame.mixer.music.get_busy():
+			while mixer.music.get_busy():
 				pass
 			if (Path(fileName).is_file()):
```

Comment:
If you're going to delete this file, then make sure all of your state is consistent. (e.g. update `self.mp3Files`)